AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreConfigurationTests.swift

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreConfigurationTests: XCTestCase {
+
+    func testConfigureWithSameSchemaDoesNotDeleteDatabase() throws {
+        Amplify.reset()
+
+        let prevoisVersion = "123"
+
+        let saveSuccess = expectation(description: "Save was successful")
+
```

Comment:
nit: you can reduce the line number where it makes sense, like within the "arrange' part of the test, you probably don't need a new line between these variables, and then add a new line between the "arrange" part and "act" part, and a newline between the "act" and "assert" sections

Reply:
True. I toke a look at other test files and it looks better. Updated